### PR TITLE
[functionapps] Improvements of devops-pipeline command

### PR DESCRIPTION
**Pull request by Hazhzeng**
### This change is made based on the feedback of 4/2019 Node report meeting
1. Add log streaming feature to show Azure DevOps progress when waiting for build.
2. Remove the unnecessary "--use_local_settings" flag and warn user in HISTORY.rst because currently the azure-functions-devops-build library does not support it.
3. Improve Azure DevOps release detection to prevent creation of duplicate

### Miscellaneous
1. Fix info/warn messages including "Azure DevOps" spelling and wording.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_provider.py [hunk 1]
```diff
@@ -281,6 +281,22 @@ def list_build_objects(self, organization_name, project_name):
         )
         return builder_manager.list_builds()
 
+    def get_build_logs_status(self, organization_name, project_name, build_id):
```
Comment (Hazhzeng) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_provider.py]: Add a feature to support Azure DevOps log streaming

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py [hunk 2]
```diff
@@ -268,36 +271,22 @@ def process_project(self):
...
+        self.logger.warning("To view your Azure DevOps project, "
+                            "please visit https://dev.azure.com/{org}/{proj}".format(
+                                org=self.organization_name,
+                                proj=self.project_name
+                            ))
+
     def process_yaml_local(self):
         """Helper to create the local azure-pipelines.yml file"""
-        # Try and get what the app settings are
-        with open('local.settings.json') as f:
```
Comment (Hazhzeng) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py]: Remove unnecessary **local.settings.json** detection and **use_local_settings** detection

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py [hunk 3]
```diff
@@ -607,34 +597,46 @@ def process_build(self, scenario):
...
         self.logger.warning("To follow the build process go to {url}".format(url=url))
 
-    def process_release(self):
-        # wait for artifacts / build to complete
-        counter = 0
+    def wait_for_build(self):
         build = None
+        prev_log_status = None
+
+        self.logger.info("========== Build Log ==========")
```
Comment (Hazhzeng) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py]: Integrating log streaming when waiting for the build

File: src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py [hunk 4]
```diff
@@ -653,31 +656,27 @@ def process_release(self):
...
-        url = (
-            "https://dev.azure.com/{org}/{proj}/_releaseProgress?_a"
-            "=release-environment-logs&releaseId={release_id}".format(
-                org=self.organization_name,
-                proj=self.project_name,
-                release_id=release.id
-            ))
-        self.logger.warning("To follow the release process go to {url}".format(url=url))
-        self.release = release
+        # Check if a release is automatically triggered. If not, create a new release.
```
Comment (Hazhzeng) [inline src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/azure_devops_build_iteractive.py]: Improve duplicated release detection

**Comment by ahmedelnably** [comment]
Please change all references from Azure DevOps to Azure Pipelines, this is the correct name of the product

**Pull request merged**